apps/docs/src/components/code/ExpandableLayout.tsx

```diff
@@ -41,18 +49,25 @@ export const ExpandableLayout = ({ scope, code }: ExpandableLayoutProps) => {
       />
 
       {/* Preview Section */}
-      <div
-        className={clsx(
-          "p-md pt-lg bg-bgPage border rounded-inherit",
-          isExpanded && "rounded-b-0",
-        )}
+      <HvProvider
+        themes={[pentahoPlus, ds5, ds3]}
+        theme={docsTheme}
+        colorMode={docsMode}
+        cssTheme="scoped"
       >
-        {liveError ? (
-          <div className="text-negative">{liveError}</div>
-        ) : (
-          <div>{element}</div>
-        )}
-      </div>
+        <div
+          className={clsx(
+            "p-md pt-lg bg-bgPage border rounded-inherit",
+            isExpanded && "rounded-b-0",
+          )}
+        >
+          {liveError ? (
+            <div className="text-negative">{liveError}</div>
+          ) : (
+            <>{element}</>
```

Comment:
why? the empty `div` is there so we don't mess up the samples when we need to change the container's layout (which we've done before multiple times). we talked about this

Reply:
we should probably abstract this away and document it properly